### ISSUE TITLE
4.9.0 MANIFEST.MF has invalid Import-Package instructions

### ISSUE THREAD

**Issue by SparxySys** [liquibase/liquibase#2672]
## Environment

**Liquibase Version**: 4.9.0

**Liquibase Integration & Version**: maven

**Liquibase Extension(s) & Version**: none

**Database Vendor & Version**: n/a

**Operating System Type & Version**: n/a

## Description
The MANIFEST.MF file inside liquibase-core-4.9.0.jar is not valid. It has Import-Package instructions for liquibase packages that require a version range of [4.9.0,1) but this cannot be satisfied: a version >= 4.9.0 and !>= 1.0.0 is not possible to satisfy.

## Steps To Reproduce

To see the contents of the file:

- download liquibase-core-4.9.0.jar from Maven Central
- extract/unpack files inside liquibase-core-4.9.0.jar
- open META-INF/MANIFEST.MF
- verify Import-Package statements

## Actual Behavior
Bundle exporting the `liquibase` package does not resolve, leading to OSGi bundles being unable to import any liquibase packages.

## Expected/Desired Behavior
OSGi bundles can import liquibase packages.

## Screenshots (if appropriate)
![Greenshot 2022-03-24 11 20 59](https://user-images.githubusercontent.com/831435/159895580-f075a3ee-7ae2-472f-8ff7-372f297f5fc0.png)

## Additional Context
n/a

**Comment by kataggart** [comment]
@SparxySys good timing - we are just looking at the MANIFEST.MF file right now and working another issue related to it and will likely do an updated release soon. Thanks!

**Comment by kataggart** [comment]
@SparxySys Before we do another full release, if we can produce a new MANIFEST.MF file for you, would you be available to test it to make sure it works on your end? Thanks!

**Comment by nvoxland** [comment]
I attached our liquibase-4.9.1-RC1.jar file renamed as a zip file because github requires that.

Would you be able to see if that file works for you, @SparxySys ? I'm always a bit shaky on OSGi so it would be a huge help to get external confirmation that the changes work.

[liquibase-4.9.1-RC1.zip](https://github.com/liquibase/liquibase/files/8343691/liquibase-4.9.1-RC1.zip)

**Comment by SparxySys** [comment]
I'm afraid we don't have a trivial test set-up for this ready right now, we ran into this issue while upgrading from a liquibase 3.x version to 4.x - at least the issue we were stuck on (the liquibase bundle not resolving) appears to have been resolved with RC1 👍 .

After quickly trying out the new version, we're currently seeing a liquibase.exception.UnexpectedLiquibaseException: Cannot find default log service - but right now I'm assuming this may just be related to our specific setup. I'll let you know once we get everything working.

**Comment by kataggart** [comment]
@SparxySys we were able to get a few other folks to test as well so look for this to be in a release on Monday!

**Comment by SparxySys** [comment]
Hello again,

We resolved the OSGi issues we were experiencing, however now Liquibase fails to start due to issue #2677, which we will to have to wait for a resolution for before we can continue.
But the issues regarding the manifest are resolved :) thank you for your swift help.
